lib/buffer.js

```diff
@@ -422,6 +426,16 @@ Buffer.prototype.fill = function fill(val, start, end) {
 };
 
 
+Buffer.prototype.address = function address() {
+  return binding.address(this);
```

Comment:
JS-side `instanceof Buffer` checks seem to current only happen in the `write*()` functions.
fill(), slice(), toString(), etc. don't seem to have these checks. Is that intentional?